tools/wdio/config/base.conf.mjs

```diff
@@ -38,7 +38,7 @@ export default function config () {
     framework: 'mocha',
     mochaOpts: {
       ui: 'bdd',
-      timeout: args.timeout,
+      timeout: 120000,
```

Comment:
This change was done so that our timeout CLI parameter only affects the agent request expect timeouts. Before this change, passing `-t 120000` would cause the expects and sauce browser session to timeout at 120 seconds resulting in subsequent tests failing because the sauce browser session was terminated.

Reply:
Our default test timeout is 85s. I increased the session timeout to 120s so that a single test timing out does not cause the whole session/suite to timeout.